### ISSUE TITLE
Can't use tag placeholders

### ISSUE THREAD

**Issue by scalp42** [dlackty/fluent-plugin-remote_syslog#1]
Hi @dlackty !

I have the following tag: `marathon.myapp` and I'm trying to tag it as `myapp`:

```
<match marathon.*>
  type remote_syslog
  host blabla.papertrailapp.com
  port 42000
  num_threads 2
  tag ${tag} # tried also tag_parts[2] or tag_parts.last
  hostname example.com
  output_include_time no
  output_include_tag no
  output_data_type attr:container_name,log
</match>
```

Unfortunately, this is not grabbing the 2nd part of the tag:

![](https://i.imgur.com/ISP8Oie.png)

![](https://i.imgur.com/6h8O5nO.png)

I'm also seeing:

```
015-12-08 01:27:51 +0000 [warn]: parameter 'num_threads' in <match marathon.*>
xxxxx
</match> is not used.
```

Any idea on how to have fluentd interpolate correctly `tag_parts[x]` correctly ?

Thanks in advance for the help!

**Comment by scalp42** [comment]
@dlackty apparently tags are not supported ? https://groups.google.com/d/msg/fluentd/vHC68RfIwa0/F1fwZ9HCAgAJ

Any chance you could implement it ?

**Comment by scalp42** [comment]
https://github.com/sonots/fluent-plugin-record-reformer/blob/master/lib/fluent/plugin/out_record_reformer.rb#L185-L201 might be a good lead.

**Comment by scalp42** [comment]
Looks like you can include a mixin: https://github.com/y-ken/fluent-mixin-rewrite-tag-name

**Comment by dlackty** [comment]
Closed via #2
